thirdparty/basis_universal/encoder/basisu_astc_hdr_6x6_enc.h

```diff
@@ -16,7 +16,7 @@ namespace astc_6x6_hdr
 	{
 		// Important: The Delta ITP colorspace error metric we use internally makes several assumptions about the nature of the HDR RGB inputs supplied to the encoder.
 		// This encoder computes colorspace error in the ICtCp (or more accurately the delta ITP, where CT is scaled by .5 vs. ICtCp to become T) colorspace, so getting this correct is important.
-		// By default the encoder assumes the input is in absolute luminance (in nits or candela per square meter, cd/m�), specified as positive-only linear light RGB, using the REC 709 colorspace gamut (but NOT the sRGB transfer function, i.e. linear light).
+		// By default the encoder assumes the input is in absolute luminance (in nits or candela per square meter, cd/m²), specified as positive-only linear light RGB, using the REC 709 colorspace gamut (but NOT the sRGB transfer function, i.e. linear light).
```

Comment:
There's something weird here, it's showing a wrong encoding in the "before" version again, when it should be UTF-8... :/

Reply:
Ah nevermind that's undoing the previous commit, so it actually keeps things in UTF-8.